### ISSUE TITLE
Codespace breaks when loading with new version.

### ISSUE THREAD

**Issue by zo0o0ot** [Leagify/prospect-scraper-mddb-2022#92]
Bug introduced in #90 

After updating the .NET version to 8, the codespace is not building successfully any more.  The goal is to fix the codespace.

Log:
```
Host information
----------------
OS: Ubuntu 22.04.5 LTS (stable release)
Image details: https://github.com/github/codespaces-host-images/blob/main/README.md
----------------
Configuration starting...
Cloning...
Creating container...
$ devcontainer up --id-label Type=codespaces --workspace-folder /var/lib/docker/codespacemount/workspace/prospect-scraper-mddb-2022 --mount type=bind,source=/.codespaces/agent/mount/cache,target=/vscode --user-data-folder /var/lib/docker/codespacemount/.persistedshare --container-data-folder .vscode-remote/data/Machine --container-system-data-folder /var/vscode-remote --log-level trace --log-format json --update-remote-user-uid-default never --mount-workspace-git-root false --omit-config-remote-env-from-metadata --skip-non-blocking-commands --skip-post-create --config "/var/lib/docker/codespacemount/workspace/prospect-scraper-mddb-2022/.devcontainer/devcontainer.json" --override-config /root/.codespaces/shared/merged_devcontainer.json --default-user-env-probe loginInteractiveShell --container-session-data-folder /workspaces/.codespaces/.persistedshare/devcontainers-cli/cache --secrets-file /root/.codespaces/shared/user-secrets-envs.json
[218 ms] @devcontainers/cli 0.68.0. Node.js v18.20.4. linux 6.5.0-1025-azure x64.
Error: Command failed: docker inspect --type image mcr.microsoft.com/vscode/devcontainers/dotnet:0-8.0
{"outcome":"error","message":"Command failed: docker inspect --type image mcr.microsoft.com/vscode/devcontainers/dotnet:0-8.0","description":"An error occurred setting up the container."}
    at wtA (/.codespaces/agent/bin/node_modules/@devcontainers/cli/dist/spec-node/devContainersSpecCLI.js:465:1260)
    at NH (/.codespaces/agent/bin/node_modules/@devcontainers/cli/dist/spec-node/devContainersSpecCLI.js:465:1002)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async YtA (/.codespaces/agent/bin/node_modules/@devcontainers/cli/dist/spec-node/devContainersSpecCLI.js:482:3771)
    at async eB (/.codespaces/agent/bin/node_modules/@devcontainers/cli/dist/spec-node/devContainersSpecCLI.js:482:4886)
    at async prA (/.codespaces/agent/bin/node_modules/@devcontainers/cli/dist/spec-node/devContainersSpecCLI.js:663:200)
    at async drA (/.codespaces/agent/bin/node_modules/@devcontainers/cli/dist/spec-node/devContainersSpecCLI.js:662:14706)
devcontainer process exited with exit code 1
Failed to create container.
Error: Command failed: docker inspect --type image mcr.microsoft.com/vscode/devcontainers/dotnet:0-8.0
Error code: 1302 (UnifiedContainersErrorFatalCreatingContainer)
Container creation failed.
```

**Comment by zo0o0ot** [comment]
More information:
https://learn.microsoft.com/en-us/dotnet/core/docker/container-images

> Starting with .NET 8, container images are more pragmatic in how they're differentiated. The following characteristics are used to differentiate images:

It looks like they may have made a breaking change to how they describe the docker images.

**Comment by zo0o0ot** [comment]
Fixed in #93
